tests/test_type_constructor.py

```diff
@@ -178,10 +179,11 @@ def test_construct_results(self):
                                   url="http://en.wikipedia.org/wiki/Scuderia_Ferrari",
                                   name="Ferrari", nationality="Italian")
 
+        time = Time(millis=datetime.time(20, 37, 33, 584000), time="1:37:33.584")
```

Comment:
I don't believe this is the correct construction of time, it should fail as it's not 20 hours. I guess pytest isn't automatically run given you're branched of develop? Either way the following should be correct:
```suggestion
        time = Time(millis=datetime.time(1, 37, 33, 584000), time="1:37:33.584")
```